.buildkite/pipeline.gpu_large.yml

```diff
@@ -27,7 +27,7 @@
     # --jobs 2 is necessary as we only need to have at least 2 gpus on the machine
```

Comment:
comment still says --jobs 2 is necessary

Reply:
yep I was exploring here. by removing jobs=2

I'm trying to understand where my cuda oom comes from

Reply:
added this backl